Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -1072,23 +1072,23 @@ static ErrorOr<void> read_start_of_frame(Stream& stream, JPEGLoadingContext& con
 
 static ErrorOr<void> read_quantization_table(Stream& stream, JPEGLoadingContext& context)
 {
-    i32 bytes_to_read = TRY(stream.read_value<BigEndian<u16>>()) - 2;
+    u16 bytes_to_read = TRY(stream.read_value<BigEndian<u16>>()) - 2;
```

Comment:
Idle musings, unrelated to this patch: We should extract a `ErrorOr<u16> read_chunk_size()` function that does:

```
static ErrorOr<u16> read_effective_chunk_size(Stream& stream)
{
    // The stored chunk size includes the size of `stored_size` itself.
    u16 stored_size = TRY(stream.read_value<BigEndian<u16>>());
    if (stored_size < 2)
        return Error::from_string_literal("stored chunk size too small");
    return stored_size - 2;
}
```

Reply:
I thought of doing this multiple time but never actually took the time :sweat_smile: